pkg/operator/deploy/deploy.go

```diff
@@ -108,9 +108,9 @@ func (o *operator) resources() ([]runtime.Object, error) {
 				}
 			}
 		}
-
 		results = append(results, obj)
 	}
+
```

Comment:
Please discard the whitespace changes in `deploy.go` and squash changes into one commit.